File: GH/1.jl
Prism.languages.julia = {
	'comment': {
		// support one level of nested comments
		// https://github.com/JuliaLang/julia/pull/6128
		pattern: /(^|[^\\])(?:#=(?:[^#=]|=(?!#)|#(?!=)|#=(?:[^#=]|=(?!#)|#(?!=))*=#)*=#|#.*)/,
		lookbehind: true
# Функция для вычисления факториала
function factorial(n::Int)
    if n < 0
        error("Факториал отрицательного числа не определен.")
    elseif n == 0 || n == 1
        return 1
    else
        return n * factorial(n - 1)
    end
end

# Использование функции
println("Факториал числа 5 равен ", factorial(5))

File: GH/1.m
Prism.languages.matlab = {
	'comment': [
		/%\{[\s\S]*?\}%/,
		/%.+/
% Функция для нахождения корней квадратного уравнения
function roots = quadratic_roots(a, b, c)
    discriminant = b^2 - 4*a*c;
    
    if discriminant > 0
        root1 = (-b + sqrt(discriminant)) / (2*a);
        root2 = (-b - sqrt(discriminant)) / (2*a);
        roots = [root1, root2];
    elseif discriminant == 0
        root = -b / (2*a);
        roots = [root, root];
    else
        error('Уравнение не имеет действительных корней.');
    end
end
